### DISCUSSION TITLE
糟糕的飲食

### DISCUSSION THREAD

**Discussion by gigitilto** [1101-GS4719-C/Python-Community-GS4719-C#2286] [General]
之前要處理的事比較多，壓力有點大，我藉由不斷的吃東西來發洩壓力，主要都是洋芋片，我知道這些零食頻繁的食用對健康造成危害，但我仍停不下口，而每天不斷的吃也讓我感到身體越來越疲累，排便不順暢，體重也有所增加。最近繁雜的事終於沒了，但我似乎也已經洋芋片成癮，有什麼辦法可以戒掉或是透過什麼其他的方式不戒掉的情況下減小不良的影響？

**Comment by kzx-creat** [comment]
改吃口香糖或堅果水果乾這類的零食，應該會有幫助?但有些果乾會另外加糖，所以要健康的話需要注意一下。
話說，我睡不著或壓力比較大的時候也會爆吃，上次是半夜睡不著爬起來吃堅果，默默就吃掉1/3包.....大包的那種

**Comment by wcyeee** [comment]
我覺得可以看看能不能改掉壓力大就吃的習慣，也許可以改去運動嗎？畢竟就算把暴食的東西改成生菜或小黃瓜對身體也沒有比較好...過量的食物對身體都會造成不良的影響的！

**Comment by christaaah** [comment]
我覺得很多時候是嘴巴想要嚼東西，而不是真正想要吃進去東西，所以我覺得可以從吃口香糖開始，或是早一點刷牙，但是如果真的想要吃東西的話，那就吃一點健康的東西，這樣影響就沒有垃圾食物那麼大了

**Comment by DarsooK** [comment]
我認為現在已經意識到問題是很好的開始，以下是一些建議：

1.逐步減少：不是一下子完全戒掉，而是逐漸減少洋芋片的攝入，比如每次吃的量減半，或設定固定的食用次數。
2.尋找替代品：選擇更健康的零食，例如堅果、水果或全穀物零食，滿足口感的同時減少負面影響。
3.規劃進食時間：限制吃零食的時段，避免隨時拿起洋芋片。
4.找到新的壓力出口：運動、冥想或寫日記都是有效的方式，幫助舒緩壓力而非依賴食物。
5.保持飲食均衡：保證正餐營養均衡，減少對零食的渴望。

如果短期內無法完全戒除洋芋片，至少可以搭配多喝水和適量運動，減小對身體的負面影響。慢慢來，習慣是可以被改變的！

**Comment by Mminniehuang** [comment]
我是會一直吃辣，對我來說吃辣是消除壓力的一大方式，辣會讓我暫時忘記煩惱，甚至吃辣吃到吃任何東西都需要辣，沒有辣就會覺得沒胃口，簡單來說就是辣成癮。
但我的腸胃真的不太好，所以每次吃太辣都會不舒服，當然每次都會在廁所反省自己不要再吃那麼辣（雖然都是說說而已）
這樣算是一種糟糕的飲食嗎😮‍💨

**Comment by qiu0109** [comment]
最近到了晚上尤其是過了12.就會變得好餓，很想吃鹹酥雞那類的東西，可是只要想到要在寒冷的天氣出門就覺得好懶，不如早點睡覺，所以我覺得睡覺跟懶應該是幫助我避免糟糕飲食的兩大重要因素。

**Comment by yue083139** [comment]
我平常是不太會想吃零食的人，但是只要有時候晚上需要小熬個夜讀書之類的就難免會感到肚子餓，在這種時候我覺得可以通過適當的飲水來緩解飢餓的感覺，或是乾脆就直接上床睡覺了，但是如果真的很餓的話吃點少量的零食倒也無可厚非，或許可以規定限制自己能吃的份量以達到逐漸減少零食的攝取。

**Comment by 113607514** [comment]
我知道洋芋片對身體不好，但壓力大時，真的很難不一口接一口，現在雖然事情少了，但好像已經對它上癮了，真的不知道該怎麼辦。

**Comment by ed112601513** [comment]
對於洋芋片成癮的問題，我覺得最簡單的方法是不要看到它和不要想到它，具體方法像是到便利商店時不要去看洋芋片那區，就算看到了不要買就好了，然後平常多做點新鮮事來轉移注意力，避免自己對洋芋片的依賴越來越重

**Comment by Taoyi-Chiang** [comment]
之前有段時間習慣一邊吃精緻澱粉一邊工作，覺得很紓壓。
後來也是發現好像真的會胖很快，超可怕。所以改成咬口香糖，讓嘴巴有東西可以咬，不會無聊。
我自己覺得滿有幫助的，可以戒掉邊工作邊吃東西的習慣🤔

**Comment by BOSENCHANG2004** [comment]
糟糕的飲食可能看起來只是生活中的小事，但長期下來對健康的影響其實非常大。可能帶來的健康問題包括:
慢性疾病風險上升、肥胖與新陳代謝失調、消化與免疫系統問題

**Comment by lichilun** [comment]
我覺得改變飲食別一次改太多，從小地方開始，比如少喝含糖飲料、多吃天然食物，慢慢調整習慣。平常多喝水、少吃加工食品，烹飪時盡量用蒸、煮、烤的方式代替油炸。設定簡單的目標，不要極端節食，偶爾放縱沒關係，重點是整體的平衡！

**Comment by Piyenparty** [comment]
我也是! 上週期中考的前一個禮拜，壓力大一直吃消夜，到現在完全戒不掉，因為忙又下雨，久沒運動，變超肥

**Comment by clairechien0627** [comment]
我上大學後也發現自己的飲食習慣變得很怪，以前喜歡喝很苦的茶或是無糖的茶，現在則是一律加糖，我現在甚至吃不了泡麵，一律覺得有湯的泡麵都很噁，到現在都還是未解之謎，或許是因為大一時經常懶得吃飯的關係 ??? (好像也完全扯不上關係) 目前再嘗試使用規律的飲食讓自己變正常一點，但好像也沒什麼用

**Comment by hollowha** [comment]
但說實在，與其你壓力大到心理生病，不如好好吃爽的，我自己的經歷覺得心理健康有點不太可逆，萬一超出你負荷的上限可能後果蠻可怕的。適當的肥胖換來快樂我覺得沒啥問題，主要讓自己身體健康就好，記得多運動健身，之前有看過一本書快樂的秘訣就是多運動，讚讚，不必一位追求社會病態的瘦，健康最重要
